From 2aa817c9b195922d77538506c9fbd99556ae6803 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Sat, 10 Aug 2024 18:25:10 +0300
Subject: [PATCH] [FSW-2704] fs1: add uart char info

this patch add user information of each character
prints over uart by boot rom.

Change-Id: I16d89acde44c1681ca9e9b756054561532fb83c7
---

diff --git a/arch/fs1/arc/dev_uart.h b/arch/fs1/arc/dev_uart.h
index 6ff8a0c..4f9448c 100644
--- a/arch/fs1/arc/dev_uart.h
+++ b/arch/fs1/arc/dev_uart.h
@@ -41,6 +41,8 @@
 /*
  * System status over UART
  * h - System in halt state
+ * r - BOOT ROM in Recovery
+ * p - BOOT ROM handover to PPBOOT
  */
 void boot_rom_putc(const char ch);
 
